lib/jekyll/commands/new.rb

```diff
@@ -114,6 +116,30 @@ def site_template
         def scaffold_path
           "_posts/0000-00-00-welcome-to-jekyll.markdown.erb"
         end
+
+        def bundle_install(path)
+          Jekyll::External.require_with_graceful_fail "bundler"
+          $stdout.print "\nRun 'bundle install' with default theme? (Y/N)  "
+          @ans = ENV["CI"] ? "N" : $stdin.gets.chomp
+          until @ans == "Y" || @ans == "N"
```

Comment:
A standard CLI must understand all cases of "y" and "yes" and "n" and "no", "y/n" is not good enough here.  I often type yes, rather than y, I am a very explicit person in what my intentions are, I'm not a keyboard basher.  I would be fuming if it rejected "yes" and probably start a Twitter storm.